app/indexers/uri_to_string_behavior.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module UriToStringBehavior
+  extend ActiveSupport::Concern
+
+  LABEL = "http://www.w3.org/2004/02/skos/core#prefLabel"
+
+  # Retrieves a value for a given URI.
+  #
+  # @param value [String] the value to retrieve. If this value starts with 'http', it is treated as a URI.
+  # @return [String]
+  #
+  # @example
+  #   uri_to_value_for('http://example.com') #=> "Failed to load RDF data: ..."
+  #   uri_to_value_for('http://id.loc.gov/authorities/names/n2017180154') #=> "University of Tennessee"
+  #   uri_to_value_for('Doe, John') #=> "Doe, John"
+  def uri_to_value_for(value)
+    return value.map { |v| uri_to_value_for(v) } if value.is_a?(Enumerable)
+    return if value.blank?
+    return value unless value.is_a?(String)
+    return value unless value.start_with?('http')
+
+    uri = value
+
+    begin
+      graph = RDF::Graph.load(uri)
+    rescue StandardError => e
+      return "Failed to load RDF data: #{e.message}"
```

Comment:
Wondering what view handling we have for URLs?  My assumption is what we return will be kind of ugly for the UI (which is where they would see this).  Should the return value be `"#{uri} (Unable to fetch label)"`?  Do we want this exception logged?  Do we want to consider specific exceptions?  I assume a not in an RDF graph to be one kind, another being a 404 from the server.  I can see different reporting.

What's the logic if the URI is something like `https://scientist.com`?  I assume we'll return something like: "Failed to load RDF data: ...https://scientist.com..."

Would it be adequate to return the URI?  Or return `"#{URL} (Failed to retrieve label)`?

Reply:
That sounds good to me, I originally wanted something very obvious to indicate something failed but included the URI in there is a good call.

Reply:
This is the error it would throw for context
```rb
indexer.uri_to_value_for('https://www.scientist.com/')

=> "Failed to load RDF data: unknown RDF format: {:base_uri=>\"https://www.scientist.com/\", :content_type=>\"text/html\", :file_name=>\"https://www.scientist.com/\"}\nThis may be resolved with a require of the 'linkeddata' gem."
```